.eslintrc

```diff
@@ -1,2 +1,4 @@
 # We use JavaScript Standard Style (https://github.com/feross/standard)
 extends: standard
+rules:
```

Comment:
You can use the ratchet mechanism that we used in ripple-lib so that you only have to fix errors in files that you modify, which makes these situations a lot less painful: https://github.com/ripple/ripple-lib/blob/develop/scripts/ci.sh#L22